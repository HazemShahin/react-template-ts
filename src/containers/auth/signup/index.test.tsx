import * as React from 'react';
import { shallow } from 'enzyme';

import { Signup } from '.';

jest.mock('react-redux', () => ({
	connect: (): jest.Mock => jest.fn(),
	useDispatch: (): jest.Mock => jest.fn()
}));

jest.mock('@utilities/hooks', () => ({
	useAppSelector: jest.fn()
}));

describe('Signup component', () => {
	it('should render successfully', () => {
		const tree = shallow(<Signup />);

		expect(tree).toMatchSnapshot();
	});
});
