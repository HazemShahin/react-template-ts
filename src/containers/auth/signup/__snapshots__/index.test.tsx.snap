// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signup component should render successfully 1`] = `
<Component
  className="o-wrapper--fancy"
>
  <Component
    onSubmit={[Function]}
  >
    <p
      className="c-form__hint"
    >
      Already have an account? 
      <Link
        to="/login"
      >
        Login
      </Link>
    </p>
  </Component>
</Component>
`;
